recipe/meta.yaml

```diff
@@ -65,18 +49,18 @@ requirements:
     - bison
   host:
     - readline
-    - ncurses
   run:
     - readline
-    - ncurses
```

Comment:
why remove this?  We still have the `--with-curses` option above?

Reply:
It does not do anything. That can be seen by the fact that the curses library is not linked to any of the final executables in the overlinking checks.

It is readline that links to curses, so it is a transitive dependency.

My best guess is that either this is a redundant or unsupported option or the build system is clever enough to know that readline will bring ncurses with it.

Reply:
Feel free to remove `--with-curses` too. The clearer the better!

Reply:
I'll let you remove it, as this is your PR. Thanks!  

It actually does do something: 

```
--with-curses

    Use the curses library instead of the termcap library. This should be supplied if your system 
    has an inadequate or incomplete termcap database.
```

from: https://www.gnu.org/software/bash/manual/html_node/Optional-Features.html, so I was assuming that this is because you wanted to use termcap instead.

Reply:
If a library is *not* linked to then it is impossible for it to do *anything*. So the option is either not working or being ignored.

Reply:
No, it has nothing to do with me wanting to use termcap vs ncurses. Simply ncurses is not used therefore we do not declare it as a dependency.

Reply:
Either way, if it is supposed to do something here it does not, so both the flag and the dependency should be removed until that is investigated (if anyone cares, main main thrust was to get Bash 5 for macOS updated since Apple only provide 3).

Reply:
Yeah, I think this this can go in once the `--with-curses` switch is removed then